### PR TITLE
Add <samp>, <small>, and <var>

### PR DESCRIPTION
**Pull request by petele**

### CODE REVIEW

File: features/small.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <small>
-description: The `<small>` element TODO.
+description: The `<small>` element represents side-comments and small print, like copyright and legal text, independent of its styled presentation.
```
Comment (foolip) [inline features/small.yml:2]: Can you also say that it makes text smaller?

File: features/samp.yml [hunk 2]
```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <samp>
-description: The `<samp>` element TODO.
+description: The `<samp>` element is used to enclose inline text which represents sample (or quoted) output from a computer program.
```
Comment (foolip) [inline features/samp.yml:2]: Can you say what the default styling is? Monospace.

Comment (ddbeck) [reply features/samp.yml:2]: I think it'd be nice to have a conventional way to describe default styling. Copying myself from another PR, but how about this pattern: _Typically styled as_ [description] _by default_.

(And [avoiding is used to](https://github.com/web-platform-dx/web-features/blob/fbd5bca14baac3c2312d230d58deedd541b96139/docs/README.md#is-used-to), sorry I will be a broken record about this.)

```suggestion
description: The `<samp>` element represents an in-line sample (or quoted) output from a computer program. Typically styled as monospaced type by default.
```

File: features/small.yml (hunk 1 above)
Comment (ddbeck) [reply features/small.yml:2]:
```suggestion
description: The `<small>` element represents side-comments and small print, like copyright and legal text, independent of its styled presentation. Typically styled in a reduced font size by default.
```

**Comment by petele** [comment]
updated based on feedback

**Review by ddbeck — APPROVED** [review]

**Pull request merged**